docs/documentation/user.md

```diff
@@ -39,22 +34,63 @@ form `/help/foo`; with special cases for `/help/` going to `index.md` and
 `/help/unknown_article` going to `missing.md` (with a 404 response). Images
 are usually linked from `static/images/help/`.
 
-This means that you can contribute to the Zulip user documentation by just
-adding to or editing the collection of Markdown files under
+This means that you can contribute to the Zulip help center documentation
+by just adding to or editing the collection of Markdown files under
 `templates/zerver/help`. If you have the Zulip development environment
 set up, you simply need to reload your browser on
 `http://localhost:9991/help/foo` to see the latest version of `foo.md`
 rendered.
 
+This system is designed to make writing and maintaining such documentation
+highly efficient. We link to the docs extensively from the landing pages and
+in-product, so it's important to keep the docs up to date.
+
 ## Writing documentation
 
 Writing documentation is a different form of writing than most people have
 experience with.
 
-Tips for adding a new article:
+### Getting started
+
+There are over 100 feature articles and longer guides in the
+[Zulip help center](https://zulip.com/help/), so use that as a resource
+and guide as you begin.
+
+- Use the list on [Zulip help center home](https://zulip.com/help/)
+  to find the section of the docs (e.g. Display settings, Sending
+  messages, Reading messages, etc.) that relates to the new feature
+  you're documenting.
+
+- Read through the existing articles in that section and pay attention
+  to the writing style and [voice](#voice), as well as any
+  [features](#documentation-features) used to enchance the readability
+  of the documentation.
+
+- Could the feature you're documenting be added or merged into an
```

Comment:
s/Could/Should/?  It's possible we should add a sentence of the form "If the new feature is a refinement on a feature that already has a dedicated page, it's likely more discoverable as a new section on that page than as an independent page, so that users can find the information by scrolling."